source/optouts/OptOutsService.cs

```diff
@@ -0,0 +1,157 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Web;
+using com.esendex.sdk.core;
+using com.esendex.sdk.exceptions;
+using com.esendex.sdk.optouts.models;
+using com.esendex.sdk.utilities;
+
+namespace com.esendex.sdk.optouts
+{
+    public class OptOutsService
+    {
+        private const string OPTOUTS_BASE_URL = "https://api.esendex.com";
+
+        private readonly Uri _baseUrl;
+        private readonly EsendexCredentials _credentials;
+
+        internal OptOutsService(string url, EsendexCredentials esendexCredentials)
+        {
+            _baseUrl = new Uri(url);
+            _credentials = esendexCredentials;
+        }
+
+        public OptOutsService(EsendexCredentials credentials) : this(OPTOUTS_BASE_URL, credentials)
+        {
+        }
+
+        public OptOutsService(string username, string password) : this(OPTOUTS_BASE_URL, new EsendexCredentials(username, password))
+        {
+        }
+
+        public OptOut GetById(Guid optOutId)

```

Comment:
have we stopped documenting with xml comments? most of the rest of the sdk is covered